pipelines/optimus/Optimus.wdl

```diff
@@ -75,7 +72,7 @@ workflow Optimus {
           i1_fastq = non_optional_i1_fastq[index],
           r2_unmapped_bam = FastqToUBam.bam_output,
           whitelist = whitelist,
-          preemptible = preemptible
+          preemptible = no_preemptible
```

Comment:
I don't see a no_preemptible variable set anywhere?

Reply:
Good catch, Cromwell just reported it too...  😢 will fix right away.

Reply:
Also I will remove all of the `preemptible = preemptible`